pkg/usageStats/stats.go

```diff
@@ -347,41 +353,47 @@ func FlushStatsToFile(orgid uint64) error {
 			atomic.StoreUint64(&ustats[orgid].BytesCount, 0)
 			atomic.StoreUint64(&ustats[orgid].LogLinesCount, 0)
 			atomic.StoreUint64(&ustats[orgid].MetricsDatapointsCount, 0)
+			atomic.StoreUint64(&ustats[orgid].LogsBytesCount, 0)
+			atomic.StoreUint64(&ustats[orgid].MetricsBytesCount, 0)
 
 			return nil
 		}
 	}
 	return nil
 }
 
-func UpdateStats(bytesCount uint64, logLinesCount uint64, orgid uint64) {
+func UpdateStats(logsBytesCount uint64, logLinesCount uint64, orgid uint64) {
 	if _, ok := ustats[orgid]; !ok {
 		ustats[orgid] = &Stats{
 			BytesCount:         0,
 			LogLinesCount:      0,
 			TotalBytesCount:    0,
 			TotalLogLinesCount: 0,
+			LogsBytesCount:     0,
 		}
 	}
```

Comment:
nit: You can change the inside to just `ustats[orgid] = &Stats{}`; all the `Stats` fields are uint64, so they'll be 0 by default. Currently I find it a little odd that you're initializing some fields but not others (but Go initializes those others to 0 anyways)

Same comment for lines 384-389